cardano-node/src/Cardano/Chairman.hs

```diff
@@ -131,25 +132,25 @@ createConnection
   maxBlockNo
   tracer
   pInfoConfig
-  socketPath = do
-    addr <- localSocketAddrInfo socketPath
-    connectTo
-      NetworkConnectTracers {
-          nctMuxTracer       = nullTracer,
-          nctHandshakeTracer = nullTracer
-        }
-      (localInitiatorNetworkApplication
-        socketPath
-        chainsVar
-        securityParam
-        maxBlockNo
-        (showTracing tracer)
-        nullTracer
-        nullTracer
-        pInfoConfig)
-      Nothing
-      addr
-    `catch` handleMuxError tracer chainsVar socketPath
+  socketPath = withIOManager $ \iocp -> do
```

Comment:
The same here, `withIOManager` must be called when chairman starts and then `iocp :: AssociateWithIOCP` should be passed down.